prow/plugins.yaml

```diff
@@ -288,6 +288,10 @@ plugins:
   - approve
 
   kubernetes/dns:
+  - approve
+  - assign
+  - hold
+  - lgtm
```

Comment:
ref https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml#L413-L431

plugins like assign/hold/lgtm are enabled org level

Reply:
you might want also add blunderbuss for auto requesting reviewers

Reply:
Thanks! Good to know. Should be fixed now.

Reply:
I think we can live without blunderbuss at the moment :) (The OWNERS file is a bit messed up at the moment)